docs/coding/recipes/balance-bounds.md

```diff
@@ -0,0 +1,103 @@
+
+There must also be a designated control account.
+
+As you can see below, this account will never actually take control of the target account's funds,
+but we will set up simultaneous transfers in and out of the control account to apply the limit.
+
+This account must have the opposite limit applied as the target account:
+
+- [`flags.credits_must_not_exceed_debits`](../../reference/account.md#flagscredits_must_not_exceed_debits)
+  if the target account has a [credit balance](../data-modeling.md#credit-balances)
+- [`flags.debits_must_not_exceed_credits`](../../reference/account.md#flagsdebits_must_not_exceed_credits)
+  if the target account has a [debit balance](../data-modeling.md#debit-balances)
+
+3. Create an Operator Account
+
+The operator account will be used to fund the Control Account.
+
+## Executing a Transfer with a Balance Bounds Check
+
+This consists of 5 [linked transfers](../../reference/requests/README.md#linked-events).
+
+We will refer to two amounts:
+
+- The **limit amount** is upper bound we want to maintain on the target account's balance.
+- The **transfer amount** is the amount we want to transfer if and only if the target account's
+  balance is within the bounds.
+
+### If the Target Account Has a Credit Balance
+
+In this case, we are keeping the Destination Account's balance between the bounds.
+
+| Transfer | Debit Account | Credit Account | Amount   | Pending ID | Flags (Note: `\|` sets multiple flags)                                                                                                                                                                  |
+| -------- | ------------- | -------------- | -------- | ---------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 1        | Source        | Destination    | Transfer | `0`        | [`flags.linked`](../../reference/transfer.md#flagslinked)                                                                                                                                               |
+| 2        | Control       | Operator       | Limit    | `0`        | [`flags.linked`](../../reference/transfer.md#flagslinked)                                                                                                                                               |
+| 3        | Destination   | Control        | `0`      | `0`        | [`flags.balancing_debit`](../../reference/transfer.md#flagsbalancing_debit) \| [`flags.pending`](../../reference/transfer.md#flagspending) \| [`flags.linked`](../../reference/transfer.md#flagslinked) |
+| 4        | `0`           | `0`            | `0`      | `3`\*      | [`flags.void_pending_transfer`](../../reference/transfer.md#flagsvoid_pending_transfer) \| [`flags.linked`](../../reference/transfer.md#flagslinked)                                                    |
```

Comment:
Nit, but if `linked` is always listed first then it will align down the columns :stuck_out_tongue: